Gemfile

```diff
@@ -43,10 +43,11 @@ group :development, :test do
 end
 
 group :test do
-  gem "capybara-webkit", ">= 1.2.0"
+  #gem "capybara-webkit", ">= 1.2.0"
```

Comment:
I don't care in this project just an FYI, we don't ever leave things like this commented in a codebase. Git keeps a history for us.